controllers/taloscontrolplane_controller.go

```diff
@@ -185,13 +181,27 @@ func (r *TalosControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		r.Log.Info("successfully updated control plane status")
 	}()
 
+	// nb: we moved the deletion reconcile before the defer to avoid additional, unneccessary patching.
```

Comment:
comment doesn't make sense any more, it's now _after_ defer ?

Reply:
yeah, right. Thanks